src/game/area.c

```diff
@@ -380,7 +380,7 @@ void play_transition_after_delay(s16 transType, s16 time, u8 red, u8 green, u8 b
 }
 
 void render_game(void) {
-    u32 first = profiler_get_delta(PROFILER_DELTA_COLLISION);
+    UNUSED u32 first = profiler_get_delta(PROFILER_DELTA_COLLISION);
```

Comment:
Yeah, the alternative is two extra lines per use of this.

Reply:
ahhh, then Arthur's suggestion about making a macro of this makes more sense to me. if debugging was disabled it could be a total noop altogether, otherwise define `first`

Reply:
https://github.com/HackerN64/HackerSM64/pull/362#discussion_r987565019

Reply:
donezo